doc/install.txt

```diff
@@ -15,12 +15,10 @@ the local system:
 Note that this needs the Python development and a C compiler installed.
 For example on Debian Linux systems use ``sudo apt-get install python-dev``.
 
-
 Setup for Windows
 -----------------
 Execute ``LinkChecker-x.y.exe`` and follow the instructions.
```

Comment:
Replace with:

Nobody is currently offering a "exe" download for Windows
See https://github.com/linkchecker/linkchecker/issues/53